app/components/views/GetStartedPage/hooks.js

```diff
@@ -254,7 +254,13 @@ export const useGetStarted = () => {
         const { isCreateNewWallet, passPhrase } = context;
         // isCreateNewWallet needs to be false for indicating a wallet
         // restore. Can be other cases if it is null or undefined.
-        const { isWatchingOnly, isTrezor } = context.selectedWallet;
+        const { selectedWallet } = context;
+        let { isWatchingOnly, isTrezor } = selectedWallet;
+        const val = selectedWallet.value;
+        if (val) {
+          if (!isWatchingOnly) isWatchingOnly = val.isWatchingOnly;
+          if (!isTrezor) isTrezor = val.isTrezor;
+        }
```

Comment:
It seems that these values are in `context.selectedWallet.value` when creating a new wallet, but in `context.selectedWallet` when opening an already created wallet. A bit confusing...